src/wallet/db.cpp

```diff
@@ -554,7 +554,6 @@ bool CDB::Rewrite(CWalletDBWrapper& dbw, const char* pszSkip)
         }
         MilliSleep(100);
     }
-    return false;
```

Comment:
Agreed, though cant say I care *too* strongly. Its a smaller/simpler change not to.